### ISSUE TITLE
Proposition to use the same search as used by Yarn

### ISSUE THREAD

**Issue by Haroenv** [Microsoft/TypeSearch#22]
Hey 👋 

I'm working at Algolia, and made the search on Yarn (https://yarn.pm). We offer this search to anyone who wants or has a use case. I saw this site and thought it could be done with this index and avoid having to download everything on page load and do the search all frontend. 

An example of the data available per package is visible [here](https://ofcncog2cu-dsn.algolia.net/1/indexes/npm-search/%40types%2Falgoliasearch?x-algolia-agent=Algolia%20for%20vanilla%20JavaScript%20(lite)%203.24.9%3Breact-instantsearch%204.4.1%3BJS%20Helper%202.23.2&x-algolia-application-id=OFCNCOG2CU&x-algolia-api-key=f54e21fa3a2a0160595bb058179bfb1e), but of course not everything is necessary to show here. 

Are you interested in this? Please let me know, and have a great day!

**Comment by karlhorky** [comment]
I guess @orta's PR to get types into the index used by npm could be relevant here?

https://github.com/algolia/npm-search/pull/346

**Comment by orta** [comment]
Yep, likely that I'll nuke this project and build something better into the definitely typed parts of the typescript website on top of the nom-search API
